projects/igniteui-angular/src/lib/tabs/tabs/tabs.component.ts

```diff
@@ -320,10 +320,18 @@ export class IgxTabsComponent extends IgxTabsDirective implements AfterViewInit,
 
         if (itemsContainerChildrenCount > 1) {
             const lastTab = this.itemsContainer.nativeElement.children[itemsContainerChildrenCount - 1] as HTMLElement;
-            itemsContainerWidth = lastTab.offsetLeft + lastTab.offsetWidth;
+            itemsContainerWidth = this.getElementOffset(lastTab) + lastTab.offsetWidth;
         }
 
         return itemsContainerWidth;
     }
+
+    private getDirection(offset: number): number {

```

Comment:
I would suggest to rename this method:
```suggestion
    private getOffset(offset: number): number {
```